job-controller/src/main/java/feast/jobcontroller/service/JobService.java

```diff
@@ -112,11 +112,6 @@ public ListIngestionJobsResponse listJobs(ListIngestionJobsRequest request) {
     List<IngestionJobProto.IngestionJob> ingestJobs = new ArrayList<>();
     for (String jobId : matchingJobIds) {
       Job job = this.jobRepository.findById(jobId).orElseThrow();
-      // job that failed on start won't be converted toProto successfully
```

Comment:
Have this `job that failed on start won't be converted toProto successfully` been fixed?